Curs04/src/views/index.js

```diff
@@ -1,8 +1,96 @@
 
 window.onload = function(){
-    const result = [];
-    result.push(LoginPage({}))
     const app = document.querySelector('#app');
+    // call LoginPage function and display LoginPage using innerHTML method
     app.innerHTML = LoginPage();
+
+    // get the form by his ID
+   const loginFormSection = document.getElementById("form-login")
+
+    const myObj = {}
```

Comment:
shouldn't this object be reset whenever you are opening a new evaluation page to make sure data from the previous evaluations does not persist into new ones?

Reply:
It should. I'm working on it. :)